src/card/basic/card.directive.js

```diff
@@ -180,9 +180,17 @@ angular.module('patternfly.card').directive('pfCard', function () {
         return ($scope.headTitle || $scope.showFilterInHeader());
       };
 
+      $scope.getHeaderClass = function () {
+        return (!$scope.showTitlesSeparator || $scope.showTitlesSeparator === 'true' ? 'card-pf-heading' : 'card-pf-heading-no-bottom');
+      };
```

Comment:
So I think Dave made this change in response to my code comment. What I was looking for was a scope variable that was just to to true or false so we weren't checking $scope.showTitlesSeparator === 'true' in the HTML. I am fine with the ng-class being in the HTML